google/resource_compute_instance_test.go

```diff
@@ -282,6 +282,41 @@ func TestAccComputeInstance_diskEncryption(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstance_kmsDiskEncryption(t *testing.T) {
+	t.Parallel()
+
+	var instance compute.Instance
+	var instanceName = fmt.Sprintf("instance-test-%s", acctest.RandString(10))
+	bootKmsKeyName := "projects/project/locations/us-central1/keyRings/testing-cloud-kms/cryptoKeys/key-0/cryptoKeyVersions/1"
+	diskNameToEncryptionKey := map[string]*compute.CustomerEncryptionKey{
+		fmt.Sprintf("instance-testd-%s", acctest.RandString(10)): {
+			KmsKeyName: "projects/project/locations/us-central1/keyRings/testing-cloud-kms/cryptoKeys/key-1/cryptoKeyVersions/1",
```

Comment:
The acceptance tests create real resources, which means that their dependencies (in this case, kms key names) also have to correspond to real resources. There's some information about running tests at https://github.com/terraform-providers/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests if you'd like to be able to run them yourself.

Reply:
In the case of KMS resources specifically, you can use https://github.com/terraform-providers/terraform-provider-google/blob/master/google/bootstrap_utils_test.go#L31. Take a look at https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_compute_disk_test.go#L316 for example usage.

Reply:
Ah, This is why the tests were running so fast 😅`go test` wasn't actually doing anything. I'll work on this.